.env.template

```diff
@@ -4,9 +4,7 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
-TRELLO_API_KEY=enter-value
-TRELLO_API_TOKEN=enter-value
-TRELLO_BOARD_ID=enter-value
-TRELLO_TODO_LIST_ID=enter-value
-TRELLO_DOING_LIST_ID=enter-value
-TRELLO_DONE_LIST_ID=enter-value
\ No newline at end of file
+MONGO_USER_NAME=mongo-user-name
+MONGO_PASSWORD=mongo-password
+MONGO_HOST=mongo-host
+MONGO_DEFAULT_DATABASE=TODO_APP
```

Comment:
Splitting these up often feels more configurable, but actually it's typically easier to just manage a single `CONNECTION_STRING` variable that encapsulates them all, as well as appropriate parameters